include/engine/guidance/assemble_geometry.hpp

```diff
@@ -70,7 +70,7 @@ inline LegGeometry assembleGeometry(const datafacade::BaseDataFacade &facade,
         cumulative_distance += current_distance;
 
         // all changes to this check have to be matched with assemble_steps
-        if (path_point.turn_instruction.type != extractor::guidance::TurnType::NoTurn)
+        if (path_point.turn_instruction.type != osrm::guidance::TurnType::NoTurn)
```

Comment:
using `guidance::TurnType` leads to an error  "`engine::guidance::TurnType` is undefined type". To avoid ambiguity `osrm::guidance::TurnType` must be used here.